dev/snippets/lib/snippets.dart

```diff
@@ -227,7 +227,8 @@ class SnippetGenerator {
       if (match != null) { // If we saw the start or end of a code block
         inCodeBlock = !inCodeBlock;
         if (match.namedGroup('language') != null) {
-          language = match[1]!;
+          language = match[1];
+          language = language!;
```

Comment:
Can we use the same pattern here that was used in the other instance and insert an `assert(language != null)` between these two lines to make the intent here a little clearer?

Reply:
Yeah sure we can!